src/RealND.hpp

```diff
@@ -135,7 +137,7 @@ namespace espressopp {
       data[i] = v[i];
   }
   
-  inline RealND::RealND(const int _dim, const real v[3]) {
+  inline RealND::RealND(const int _dim, const real v[]) {
     setDimension(_dim);
```

Comment:
Nice one, Is this constructor use somewhere? 
Please add check up if number of elements in ```v``` is equal to ```_dim```.